metaflow/plugins/aws/step_functions/step_functions_cli.py

```diff
@@ -614,6 +618,83 @@ def _token_instructions(flow_name, prev_user):
         )
 
 
+@step_functions.command(help="Terminate flow execution on Step Functions.")
```

Comment:
minor nit - but for consistency `AWS Step Functions`. not blocking for merging this PR in, but would be good if you can sweep this and various CLIs to polish up the help strings and make them consistent.